templates/default/sssd.conf.erb

```diff
@@ -131,6 +131,7 @@ ldap_force_upper_case_realm = True
 ldap_group_member = msSFU30PosixMember
 
 <% else -%>
-ldap_group_member = uniqueMember
+ldap_group_member = <%= node[:authconfig][:ldap][:group_member] %>
+ldap_group_search_base = <%= node[:authconfig][:ldap][:group_search_base] %>
```

Comment:
The default for ldap_group_search_base is empty string.
Have you tested, and does sssd still properly configure and start up when an empty string is passed in for the search base?